.github/autobuild/mac.sh

```diff
@@ -68,6 +85,29 @@ pass_artifact_to_job() {
     echo "Moving build artifact to deploy/${artifact}"
     mv ./deploy/Jamulus-*installer-mac.dmg "./deploy/${artifact}"
     echo "::set-output name=artifact_1::${artifact}"
+
+    artifact2="jamulus_${JAMULUS_BUILD_VERSION}_mac${ARTIFACT_SUFFIX:-}.pkg"
+    if [ -f ./deploy/Jamulus_*.pkg ]; then
+        echo "Moving build artifact2 to deploy/${artifact2}"
+        mv ./deploy/Jamulus_*.pkg "./deploy/${artifact2}"
+        echo "::set-output name=artifact_2::${artifact2}"
+    fi
+}
+
+valid8_n_upload() {
+    echo ">>> Processing validation and upload..."
+    # test the signature of package
+    pkgutil --check-signature "${ARTIFACT_PATH}"
+    # attempt validate and then upload of pkg file, using previously-made keychain item
+    xcrun altool --validate-app -f "${ARTIFACT_PATH}" -t macos -u $NOTARIZATION_USERNAME -p $NOTARIZATION_PASSWORD
+    xcrun altool --upload-app -f "${ARTIFACT_PATH}" -t macos -u $NOTARIZATION_USERNAME -p $NOTARIZATION_PASSWORD
+
+    ## altool is deprecated - migrate soon to using notarytool eg:
```

Comment:
Why can't this be used now?

Reply:
It can, sure. But altool will work for another 12 months, and I am loathe to change the CI when it is working (as it is for me).
So I put a helpful comment for the near future. It requires some more keychain manipulation so it's not just a drop-in replacement.

Reply:
Hmm. Still shouldn't we use the more future proof way as soon as possible (prevents duplicate work and debugging if it's suddenly gone).

Reply:
In general, I agree, I'm just tight on time. I'll take a look today if I can

Reply:
Updated, now working with `notarytool`. It was easier than expected - uses the same credentials as in Notarization.

Note: the Github Action (https://github.com/devbotsxyz/xcode-notarize) used currently for dmg Notarization still uses the deprecated `altool` in the background. And that repo hasn't been updated in 2 years :/